src/components/Admin/AdminLocalisations.tsx

```diff
@@ -0,0 +1,211 @@
+import React, { useState, useRef } from 'react'
+import { Header, Button, Checkbox, Icon, ModalProps } from 'semantic-ui-react'
+import { postRequest } from '../../utils/helpers/fetchMethods'
+import config from '../../config'
+import { LanguageOption, useLanguageProvider } from '../../contexts/Localisation'
+import usePageTitle from '../../utils/hooks/usePageTitle'
+import { useToast, topLeft } from '../../contexts/Toast'
+import ModalWarning from '../Main/ModalWarning'
+import { exportLanguages } from '../../utils/localisation/exportLanguages'
+import { importLanguages } from '../../utils/localisation/importLanguages'
+
+export const AdminLocalisations: React.FC = () => {
+  const { strings, refetchLanguages, languageOptionsFull: languageOptions } = useLanguageProvider()
+  usePageTitle(strings.PAGE_TITLE_LOCALISATION)
+  const fileInputRef = useRef<any>(null)
+  const [exportDisabled, setExportDisabled] = useState(true)
+  const [importDisabled, setImportDisabled] = useState(true)
+  const showToast = useToast({ position: topLeft })
+  const [showModalWarning, setShowModalWarning] = useState<ModalProps>({ open: false })
+  const [hoverIndex, setHoverIndex] = useState<number | null>(null)
+
+  const handleSelect = async (language: LanguageOption, index: number) => {
+    const enabled = !language.enabled
+    const result = await postRequest({
+      url: `${config.serverREST}/admin/enable-language?code=${language.code}&enabled=${enabled}`,
+    })
+    if (result.success) {
+      console.log('Language updated')
```

Comment:
Useful to add to which code here - or just remove the log

Reply:
Yup, have added code: a5a9d8f9